qa/workunits/cephtool/test.sh

```diff
@@ -1682,7 +1682,7 @@ function test_mon_osd_misc()
   ceph pg set_full_ratio 95 2>$TMPFILE; check_response 'not in range' $? 22
 
   # expect "not in range" for invalid overload percentage
-  ceph osd reweight-by-utilization 80 2>$TMPFILE; check_response 'higher than 100' $? 22
+  ceph osd reweight-by-utilization 80 2>$TMPFILE; check_response 'not in range' $? 22
```

Comment:
hmm, i test it on hammer 0.94.5 "80 not valid:  80 not in range [100L]",
the master branch is fine.